app/controllers/api/events_controller.rb

```diff
@@ -41,6 +41,8 @@ def create
   # PATCH/PUT /api/events/1.json
   # PATCH/PUT /api/events/654331ae-1710-42e5-bdf4-65a03a80c614.json
   def update
+    fail ActiveRecord::RecordNotFound unless @event
```

Comment:
Why do you need those?
The find in 
https://github.com/voc/voctoweb/blob/a931b180d8f5f7e4b1164093aa787ea00e03e051/app/controllers/api/conferences_controller.rb#L63
should raise if I'm remembering correctly

Reply:
Nope, I got a 'trying to access nil' exception or similar – then I found this solution in the public api e.g. https://github.com/voc/voctoweb/blob/master/app/controllers/public/conferences_controller.rb#L17

Reply:
https://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find_by-21